lib/storage/temporary-cloud-storage.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/**
+ * ADOBE CONFIDENTIAL
+ * ___________________
```

Comment:
@sthyagarajan Could it be you missed to push some changes? There still is the duplicated license header

Reply:
@tmathern not sure but fixed it now

Reply:
Thanks!